docs/agent-oriented-approach.md

```diff
@@ -0,0 +1,49 @@
+# Agent-oriented problem solving
+In this section, we highlight some of the most fundamental characteristics of the agent-based approach to solution development, and problem solving in general, which may be different from some of the existing paradigms and methodologies people are used to. We hope to be able to guide you towards having the right mindset when designing your own agent-based solutions to real world problems. 
+
+<!-- Our aim with the following discussion is that we guide you--> 
+
+## Decentralisation
+First and foremost is decentralisation. Multi-agent systems (MAS) are inherently decentralised. The vision is an environment in which every individual is able to directly connect with everyone else and interact with them without having to rely on a third-party to facilitate this. This is in direct contrast to centralised systems in which a single entity is the central point of authority, through which all interactions happen. Systems based on the client-server model are an example of centralised systems in which <!--a central facility (the server) provides a set of services to the clients. Furthermore, in these systems,--> clients interact with one another, regarding a specific service (e.g. communication, trade), through the server. 
```

Comment:
`every agent` rather than `every individual`..although it might be individuals too it is probably easier to follow this text if we keep it focused on agents

Reply:
please remove comment

Reply:
Done